app/src/main/res/layout/quest_bike_rental_capacity.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Isn't this exactly the same layout as for the bicycle capacity quest? (If yes, why not use that layout. Actually, why not use the same form?)

Reply:
The hint in the quest form doesn't make sense for this quest type:

> Note that most normal stands can be used from both sides to park each one bike.

But the quest form can of course be reused by just hiding that hint. Done in [`844328e`](https://github.com/streetcomplete/StreetComplete/pull/4008/commits/844328e02c45dc5b331202f47726e20a558327fe).